README.md

```diff
@@ -103,6 +103,13 @@ Shortcuts this extension introduces:
 | Z, M    | Hide All Code Cells |
 | Z, R    | Show All Code Cells |
 
+### Transition between modes
+
+Note that there are three modes (Insert, Normal and Jupyter Command). The transition between them happens only with the key ESC, but you can enable a different transition from Normal mode to Jupyter Command mode.
+
+Go to *Setting / Settings Editor / Notebook Vim* and unmark the option `Enable Esc and Ctrl-[ leaving vim Normal mode to Jupyter Command mode`. From now on, `ESC` will move you from Insert to Normal mode and `Shift+ESC` will move you from Normal to Jupyter Command mode.
```

Comment:
Minor formatting suggestion:

```suggestion
Go to *Setting / Settings Editor / Notebook Vim* and unmark the option `Enable Esc and Ctrl-[ leaving vim Normal mode to Jupyter Command mode`. From now on, <kbd>Esc</kbd> will move you from Insert to Normal mode and <kbd>Shift</kbd> + <kbd>Esc</kbd> will move you from Normal to Jupyter Command mode.
```

Reply:
Though feel free to disregard though as the rest of the readme is using code markers so maybe not worth it for the sake of consistencty?

Reply:
I think it was a good recommendation, I changed it :+1: